src/xdocs/google_style.xml

```diff
@@ -648,7 +648,7 @@
                   <a href="config_blocks.html#LeftCurly">LeftCurly</a>
                   <br/>
                   <span class="wrapper inline">
-                    <img src="images/ok_green.png" alt="" />
+                    <img src="images/ok_blue.png" alt="" />
```

Comment:
How this is related with issue?:)

Reply:
https://checkstyle.org/google_style.html#Google_Legend

Green for `covers all`, blue for `covers some`.

Reply:
https://checkstyle.org/google_style.html#Google_Legend

Blue for `covers all`, green for `covers some`.